ocaml/idl/datamodel.ml

```diff
@@ -6577,6 +6579,25 @@ let vdi_disable_cbt = call
     ~allowed_roles:_R_VM_ADMIN
     ()
 
+let vdi_hidden_cbt_setter = call
```

Comment:
I think we should call the function `set_cbt_enabled` to be more in line with the rest of the function names

Reply:
There is also the Db.VDI.set_cbt_enabled function which we call though. I thought having the same name twice would be confusing

Reply:
we already have many setters of this form at the xenapi level: http://xapi-project.github.io/xen-api/classes/vdi.html
For me I think it is actually helpful that it has the same name, because that name suggests that this function does a DB write.

Reply:
I agree with @gaborigloi.